agent/ecs_client/model/api/api-2.json

```diff
@@ -679,6 +685,7 @@
       "members":{
         "name":{"shape":"String"},
         "image":{"shape":"String"},
+        "repositoryCredentials":{"shape":"RepositoryCredentials"},
```

Comment:
is this mixed with private registry work? if so, is this blocking and/or cause conflicts with any other work?

Reply:
looks like this change is just bringing in the acs model changes. it shouldnt block and im expecting it to just resolve itself during merge.